src/ST7735_t3.h

```diff
@@ -234,12 +234,35 @@ class ST7735_t3 : public Print
 
   void setAddr(uint16_t x0, uint16_t y0, uint16_t x1, uint16_t y1)
     __attribute__((always_inline)) {
-        writecommand(ST7735_CASET); // Column addr set
-        writedata16(x0+_xstart);   // XSTART 
-        writedata16(x1+_xstart);   // XEND
-        writecommand(ST7735_RASET); // Row addr set
-        writedata16(y0+_ystart);   // YSTART
-        writedata16(y1+_ystart);   // YEND
+        // Keep a copy of the last address window transmitted.
+	// Do not transmit it again until the window actually changes.
+        static uint16_t _x0 = 0xFFFF, _y0 = 0xFFFF, 
```

Comment:
This doesn't work if multiple instances of the display are connected - you need to make the stored values private class members.